tests/suites/test_suite_rsa.function

```diff
@@ -17,454 +17,6 @@
  * END_DEPENDENCIES
  */
 
-/* BEGIN_CASE depends_on:MBEDTLS_CHECK_PARAMS:!MBEDTLS_PARAM_FAILED_ALT */
-void rsa_invalid_param( )
-{
-    mbedtls_rsa_context ctx;
-    const int valid_padding = MBEDTLS_RSA_PKCS_V21;
-    const int invalid_padding = 42;
```

Comment:
There is an `invalid_padding` and an `invalid_mode`  thus there are probably some checks to preserve here, please double check.

Reply:
Yes, there is `invalid_padding` but I don't see the `invalid_mode`. I reverted the deletion and retained the `invalid_padding` checks. But should I use the `TEST_EQUAL` instead of the `TEST_INVALID_PARAM`?

Reply:
Ah, OK, the function will not run now, so it can be left as it is until the next step.

Reply:
The functions tested with `invalid_padding` do not return an error code thus we can just remove the tests. Otherwise there is an `invalid_mode` (see line 27 below) and the associated tests can probably be kept (even if some parameters are equal to NULL).

Reply:
Nope the `invalid_mode` tests have been removed by #4515 and I was looking before the merge commit probably. Thus eventually we just have to get rid of rsa_invalid_param() in tests/suites/test_suite_rsa.function.